hyperdome_client/hyperdome_client.py

```diff
@@ -33,19 +33,25 @@
 from .server_status import ServerStatus
 from .add_server_dialog import AddServerDialog
 
-import requests
 
+import socks
+import socket
+import requests
```

Comment:
```suggestion
import requests
import traceback
```
For displaying the trace in a comment below